api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -270,12 +270,8 @@ private DbUser initializeUserIfNeeded() {
 
     final Profile profile = request.getProfile();
 
-    // We don't include this check in validateAndCleanProfile since some existing user profiles
-    // may have empty addresses. So we only check this on user creation, not update.
-    Optional.ofNullable(profile.getAddress())
-        .orElseThrow(() -> new BadRequestException("Address must not be empty"));
-
-    profileService.validateAndCleanProfile(profile);
+    // Run all profile validation by passing in a null 'previous' object.
```

Comment:
nit: I'd add a wrapper around this in the service so you could do `profileService.validateProfile(profile);`. Ideally, we should very rarely need to write out `null`.